rosdep/base.yaml

```diff
@@ -1776,13 +1776,21 @@ gz-cmake2:
   ubuntu:
     focal: [libgz-cmake2-dev]
     jammy: [libgz-cmake2-dev]
+gz-cmake3:
+  ubuntu:
+    focal: [libgz-cmake3-dev]
+    jammy: [libgz-cmake3-dev]
```

Comment:
I must be missing something. Does rosdep not allow packages from `http://packages.osrfoundation.org/gazebo/ubuntu-stable` anymore?

Works locally just fine:

```bash
$ apt show libgz-cmake3-dev
Package: libgz-cmake3-dev
Version: 3.4.1-1~jammy
Priority: optional
Section: libdevel
Source: gz-cmake3
Maintainer: Jose Luis Rivero <jrivero@osrfoundation.org>
Installed-Size: 2,105 kB
Depends: cmake
Breaks: libignition-cmake3-dev (<< 2.999.999+nightly+git20220630+1rea69f3cd041c33f9c65946e71f0f199220bd8966-1)
Replaces: libignition-cmake3-dev (<< 2.999.999+nightly+git20220630+1rea69f3cd041c33f9c65946e71f0f199220bd8966-1)
Homepage: http://gazebosim.org/
Download-Size: 207 kB
APT-Manual-Installed: yes
APT-Sources: http://packages.osrfoundation.org/gazebo/ubuntu-stable jammy/main amd64 Packages
Description: Gazebo CMake Library - Development files
 CMake modules to be used by the Gazebo projects.
 .
 This package is required to build Gazebo projects, as well as to link your
 third party projects against them. It provides modules that are used to find
 dependencies of Gazebo projects and generate cmake targets for consumers of
 Gazebo projects to link against.
 ```